imap/seqset.go

```diff
@@ -26,6 +27,12 @@ func (seqval SeqVal) String() string {
 
 type SeqSet []SeqVal
```

Comment:
SeqVal has two `int`s should we use `uint32` ?

Reply:
Good catch. fixed.